cli/tools/init/templates/main_test.ts

```diff
@@ -1,4 +1,4 @@
-import { assertEquals } from "{CURRENT_STD_URL}assert/mod.ts";
+import { assertEquals } from "jsr:@std/assert";
```

Comment:
Can we use a deno.json import entry for `@std/assert` and have a version range specified in there? Maybe we could just reuse the deno add code for now

Reply:
Er, I'm not sure I understand - I can certainly move it to `deno.json`, but whole point of this PR was to not require bumping version on each release. I think we can just leave it unversioned for the time being and change it to `@^1` once we release appropriate version for `@std/assert`?

How `deno add` code is relevant here?

Reply:
Sure, that sounds good to wait until 1.0